build.gradle

```diff
@@ -31,9 +31,15 @@ gradlePlugin {
         }
     }
 }
+wrapper {
+    gradleVersion = '6.8.3'
```

Comment:
Sorry, I've too less knowledge what's the purpose of if.
Why do you need it ?

Reply:
Tried to add a gradle sources to play with

Reply:
With my current knowledge, it's just noise